scripts/spec.sh

```diff
@@ -1,17 +1,33 @@
 #!/bin/bash
-
 mkdir -p /tmp/parachain/
 
+ROCOCO_PLAIN=/tmp/parachain/rococo.plain.json
+ROCOCO=/tmp/parachain/rococo.json
+
+# ##############################################################################
+# #                                                                            #
+# #                                  PEREGRINE                                 #
+# #                                                                            #
+# ##############################################################################
+docker run parity/rococo:rococo-v1-0.8.30-943038a8-f14fa75f build-spec --chain rococo-local --disable-default-bootnode >$ROCOCO_PLAIN
```

Comment:
I would leave a space before and after the redirect symbol ">".

Reply:
I would too, but i have a weird bash formatter that removes it. 🤔 i added it. lets hope it survives the formatter for a while 😅